### PR TITLE
feat: [WC-1832] - update icons

### PR DESCRIPTION
**Pull request by gjulivan**
### Pull request type

---

### CODE REVIEW

File: packages/customWidgets/calendar-web/src/Calendar.xml
Comment (iobuhov) [inline packages/customWidgets/calendar-web/src/Calendar.xml:1]: Let's not remove this one as it still probably used in Mx8

**Review by iobuhov — CHANGES_REQUESTED** [review]
Please, keep icons in xml and bump package versions, if needed.

**Comment by iobuhov** [comment]
Tested. Works as expected.

**Comment by iobuhov** [comment]
@leonardomendix, can I get approval? 😄

**Review by leonardomendix — APPROVED** [review]

**Pull request merged**